services/net/fileupload/TNO.Services.FileUpload.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <Nullable>enable</Nullable>
+    <RootNamespace>TNO.Services.FileUpload</RootNamespace>
+    <Version>1.0.0.0</Version>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\libs\net\services\TNO.Services.csproj" />
+    <ProjectReference Include="..\..\..\libs\net\kafka\TNO.Kafka.csproj" />
```

Comment:
Does this communicate with Kafka at all?

Reply:
we don't need actually.